src/main/java/org/stocker/routes/UpdateRecordsRoute.java

```diff
@@ -69,28 +64,19 @@ public void handle(YokeRequest yokeRequest, Handler<Object> handler) {
         response.end();
     }
 
-    public void updateStocksForDate(Date dateObj) throws StockDataNotFoundForGivenDateException, NseDataObjParseException {
-        ArrayList<String> processedStocksList = new ArrayList<>();
-
-        List<NseDataObj> dataObjList;
-        dataObjList = stockDataClient.getStockData(dateObj, ReportType.BHAV_REPORT);
-        List<NseDataObj> validStocks = dataObjList.stream()
-                                        .filter(dataObj -> StockValidityCheckerService.isDataValid(dataObj)
-                                                && isValidAsPerInstrument(dataObj))
-                                        .collect(Collectors.toList());
+    public void updateStocksForDate(Date dateObj) throws StockDataNotFoundForGivenDateException, NseDataObjParseException, IOException {
+        //TODO : Find an API which will give all US stocks and use that
+        List<String> usStockSymbols = Arrays.asList("TSLA", "GOOG");
+        Calendar from = Calendar.getInstance();
+        from.setTime(dateObj);
```

Comment:
I dont think we can move away from Calendar since the library takes Calendar as parameter. Unless we make the API calls to get the data ourselves. I am making sure we use date in all our code.